### PR TITLE
Aim shooter

### PR DESCRIPTION
**Pull request by JamisOrr**

### CODE REVIEW

File: Robot2024
Comment (Gold856) [inline Robot2024:1]: Remove.

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	}
+
+	// Called when the command is initially scheduled.
+	@Override
+	public void initialize() {
+		switch (m_operation) {
+			case CALC_AND_SET:
+				double actuatorHeightSetpoint = m_targeter.calcAimerHeightFromDistance(m_distanceMeters);
+				m_aimerSubsystem.setAimerHeight(actuatorHeightSetpoint);
+				break;
```
Comment (NatalieEMann) [inline src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java:46]: you can put the breaks at the end...not necessary though.

**Review by NatalieEMann — APPROVED** [review]

**Review by Gold856 — CHANGES_REQUESTED** [review]
Adjust operations are completely broken, either fix them or remove them

File: src/main/java/frc/robot/Targeter.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
+package frc.robot;
+
+// imports
```
Comment (Gold856) [inline src/main/java/frc/robot/Targeter.java]: Remove

File: src/main/java/frc/robot/Targeter.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+public interface Targeter {
+
+	public double m_speakerHeight = Constants.AimerConstants.speakerHeight;
+	public double m_aimerLength = Constants.AimerConstants.aimerLength;
+	// aka small delta x and big delta y
+
```
Comment (Gold856) [inline src/main/java/frc/robot/Targeter.java]: Remove

**Comment by JamisOrr** [comment]
> Adjust operations are completely broken, either fix them or remove them

I removed it from the command and commented it from the subsystem, keeping it around in some form in case we ever need it -- whatever that may be

File: src/main/java/frc/robot/Targeter.java
Comment (Gold856) [inline src/main/java/frc/robot/Targeter.java:1]: ~~Remove extra targeters, we can come back to them later~~ Actually, you can leave them if you want.

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 4]
```diff
@@ -192,4 +192,29 @@ public static final class PneumaticsConstants {
...
+		public static final double kP = 0.06000; // 0.000_1; // TODO tune PID values
+		public static final double kI = 0.000;
+		public static final double kD = 0;
+		public static final double kIz = 5; // 0.0;
+		public static final double kFF = 0.0; // .000_1050;
+		public static final double kMaxOutput = 1;
+		public static final double kMinOutput = 0;
+		public static final double kGearRatio = 3;
+		public static final double speakerHeight = 2.032145; // TODO change to this height - robot height
+		public static final double aimerLength = .25; // TODO change to deltaX between edge of robot and aimer axis
```
Comment (Gold856) [inline src/main/java/frc/robot/Constants.java]: Rename to start with `k`

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Targeter.java
Comment (JamisOrr) [reply src/main/java/frc/robot/Targeter.java:1]: There is no reason to remove that code, it doesn't help anything and only raises the probability of us being mildly annoyed in the future

File: src/main/java/frc/robot/Targeter.java (hunk 3 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/Targeter.java]: Yeah

**Review by Gold856 — APPROVED** [review]

**Pull request merged**